app/views/users/new.html.erb

```diff
@@ -1,3 +1,18 @@
 <% provide(:title, t("title_signup")) %>
 <h1><%= t(".signup") %></h1>
-<p><%= t(".description") %></p>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for(@user) do |f| %>
```

Comment:
phân biệt form for và các loại khác

Reply:
+ tạo form có đối tượng (form tag thì không, form with có hay không đều được) và được dùng khi tạo mới hoặc update 1 đối tượng
+ Sử dụng form builder field helper (form tag thì không, form with sử dụng form builder cho tất cả các field)
+ Có thể tự động thêm id và class cho form và các form field (form tag chỉ thêm cho form field, form with tự động thêm id cho form, với form field thì tự thêm bằng tay)
+ các thuộc tính id và class cần được đóng gói (form tag cũng vậy, form with không cần và các form field không cần phải tương ứng với các thuộc tính của model)
+ form for và form tag không mặc định ở chế độ remote, còn form with thì có